cadence/freshmint-claim-sale/FreshmintClaimSale.cdc

```diff
@@ -330,7 +391,7 @@ pub contract FreshmintClaimSale {
         /// The integer value is the number of NFTs an  
         /// address is entitled to claim.
         ///
-        access(self) let claimsByAddress: {Address: UInt}
+        pub let claimsByAddress: {Address: UInt}
```

Comment:
🤔  Wouldn't it be safer if `claimsByAddress`  remains private? But then again I think everyone that has access to it so far is in a  controlled environment

Reply:
Yeah, I'll keep it as private. It shouldn't matter but I also can't think of a good reason to make it public right now (I'm not sure why I changed it).